README.md

```diff
@@ -1,238 +1,34 @@
-# `react-native-background-actions`
-![](https://github.com/Rapsssito/react-native-background-actions/workflows/Release/badge.svg)
-
+<p align="center">
+    <img src="https://i.imgur.com/G8BUzdZ.png" />
+    <br></br>
+    <img src="https://github.com/Rapsssito/react-native-background-actions/workflows/Release/badge.svg" />
+    <img src="https://img.shields.io/npm/dw/react-native-background-actions" />
+    <img src="https://img.shields.io/npm/v/react-native-background-actions?color=gr&label=npm%20version" />
+</p>
 
 React Native background service library for running **background tasks forever in Android & iOS**. Schedule a background job that will run your JavaScript when your app is in the background or foreground.
 
-### WARNING
+### <span style="color:Red">WARNING</span>
```

Comment:
This style is not working for me. Should we remove it?

Reply:
Yes, It doesn't work in GitHub and I forgot to remove it.